ui/admin/app/components/form/auth-method/oidc/index.hbs

```diff
@@ -6,6 +6,14 @@
   @showEditToggle={{if @model.isNew false true}}
   as |form|
 >
+
+  <form.actions
```

Comment:
This was added because the form is long and the actions fall below the fold where they may not be seen.